scripts/config/templates/nginx.conf.erb

```diff
@@ -50,6 +50,8 @@ http {
     resolver <%= resolver %>;
   <% end %>
 
+    client_max_body_size 32m;
```

Comment:
Should this be in the higher up http block? My understanding is that nested blocks inherit from above but blocks with nested ones within don't inherit from their children. If we put it in http {} then I think it's global, which might be better.

Reply:
I put it in the parent block! :)